docs/source/markdown/options/compression-level.md

```diff
@@ -0,0 +1,7 @@
+####> This option file is used in:
+####>   podman manifest push, push
+####> If file is edited, make sure the changes
+####> are applicable to all of those.
+#### **--compression-level**=*level*
+
+Specifies the compression level to use.  The value is specific to the compression algorithm used, e.g. for zstd the accepted values are in the range 1-20 (inclusive), while for gzip it is 1-9 (inclusive).
```

Comment:
What are the defaults?

Reply:
thanks.  I was waiting for the c/common change to merge before pushing the new version.

Should be addressed now